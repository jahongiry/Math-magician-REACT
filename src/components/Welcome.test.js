import React from 'react';
import renderer from 'react-test-renderer';
import Welcome from './Welcome';

it('renders if there is Welcome component', () => {
  const three = renderer.create(<Welcome />).toJSON();
  expect(three).toMatchSnapshot();
});
