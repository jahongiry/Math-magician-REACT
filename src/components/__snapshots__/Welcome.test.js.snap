// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders if there is Welcome component 1`] = `
<div
  className="container"
>
  <h2
    className="heading"
  >
    Welcome to our page!
  </h2>
  <p>
    Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
  </p>
  <p>
    Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
  </p>
</div>
`;
